server/models/Notification.js

```diff
@@ -1,21 +1,24 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 
 const notificationSchema = mongoose.Schema(
   {
     notificationType: {
       type: String,
-      enum: ["success", "warning", "info", "confirmation"],
-      default: "info",
+      enum: ['success', 'warning', 'info', 'confirmation', 'dog sitting'],
+      default: 'info',
     },
     title: { type: String, required: true },
     description: String,
     read: { type: Boolean, default: false },
     userId: {
       type: mongoose.Schema.Types.ObjectId,
-      ref: "user",
+      ref: 'user',
+    },
+    context: {
+      profilePhotoURL: String,
```

Comment:
You are on the right track here with this, my idea for this was it's an object that can have anything in it. Mongoose has a Mixed data type, this aligns with what was initially envisioned here. Hope this helps

Reply:
Essentially, we want this `context` property to be able to include anything which generalizes this for further support in notification types requiring various different contexts.